css/card.css

```diff
@@ -1,30 +1,143 @@
-.card,
-.card-front,
+:root {
+  --card-width: 200px;
+  --card-height: 279px;
+  --border-radius: 5%/3.5%;
+}
+
+tcg-card, .card-component, .card-front,
 .card-back {
-  display: inline;
+  width:var(--card-width);
+  height:var(--card-height);
+}
+tcg-card, .card-component, .card-front,
+.card-back {
+  display: inline-block;
+}
+tcg-card {
+  margin: 3%;
+  perspective: 600px; /* for the card animation */
+}
+
+#gacha-controls #card-list {
+  min-height:52vh;
+}
+#gacha-controls tcg-card {
+  --card-width: 400px; /* adjust the card size for the gacha page */
+  --card-height: 559px; /* adjust the card size for the gacha page */
+  position: absolute;
+  right:10%;
+  transition: all 0.3s ease; /* make any changes to this card flow, so it doesn't snap */
+}
+tcg-card .card-info {
+  font-size:0.75em;
+  color:white;
+  margin-top:2%;
+}
+#gacha-controls tcg-card .card-info {
+  display:none; /* don't want to see that particular card information on the gacha page */
+}
```

Comment:
can you delete these changes/ add your nice styling to o5v3's version of the card subtitle stuff?

Reply:
This has been updated to align to o5v3's updates.